authz/setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit;
+set -o nounset;
+set -o pipefail;
+set -o xtrace;
+
+source ./test/helper.sh
+test_name="authz-validation"
+setup_gke_basic "${test_name}" "${ZONE}" "${REGION}"
```

Comment:
In the interest of making tests run faster, mind switching to https://github.com/kubernetes-sigs/kubectl-validate? I know this isn't a presubmit yet, but I'd like it to get to that point in the not too distant future.

Reply:
Done.